### ISSUE TITLE
question 

### ISSUE THREAD

**Issue by weizhao-11** [iFe1er/SMU_pytorch#3]
请问yolov5 使用smu激活函数为什么会有报这种错，UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skip？

**Comment by koushik313** [comment]
According to the current Pytorch documentation, you need to apply 'lr_scheduler.step()' after "optimizer.step()" else lr_scheduler will not work. You can check the following discussion for more details.

https://discuss.pytorch.org/t/userwarning-detected-call-of-lr-scheduler-step-before-optimizer-step-in-pytorch-1-1-0-and-later-you-should-call-them-in-the-opposite-order-optimizer-step-before-lr-scheduler-step/88295
